cmd/firewall.go

```diff
@@ -173,3 +198,189 @@ func (c *firewallCmd) createRequestFromCLI() (*models.V1FirewallCreateRequest, e
 		Ips:                mcr.Ips,
 	}, nil
 }
+
+// for test
+// args = [firewall id, identity, control plane address, auth key]
+func (c *config) firewallSSH(args []string) (err error) {
+	if len(args) < 1 {
+		return fmt.Errorf("Machine ID is expected as an argument")
+	}
+	firewallID := args[0]
+
+	// Get firewall's project
+	machineGetResp, err := c.client.Machine().FindMachine(machine.NewFindMachineParams().WithID(firewallID), nil)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve firewall details: %w", err)
+	}
+	projectID := machineGetResp.Payload.Allocation.Project
+
+	authKeyResp, err := c.client.VPN().GetVPNAuthKey(vpn.NewGetVPNAuthKeyParams().WithBody(&models.V1VPNRequest{
+		Pid: projectID,
+	}), nil)
+	if err != nil {
+		return fmt.Errorf("failed to get VPN auth key: %w", err)
+	}
+
+	cli, err := dockerclient.NewClientWithOpts(dockerclient.FromEnv)
+	if err != nil {
+		return fmt.Errorf("failed to initialize Docker client: %w", err)
+	}
+
+	// Deploy tailscaled
+	ctx := context.Background()
+	if err := pullImageIfNotExists(ctx, cli, tailscaleImage); err != nil {
+		return fmt.Errorf("failed to pull tailscale image: %w", err)
+	}
+
+	containerConfig := &container.Config{
+		Image: tailscaleImage,
+		Cmd:   []string{"tailscaled"},
+	}
+	hostConfig := &container.HostConfig{
+		Binds: []string{
+			"/var/lib:/var/lib",
```

Comment:
This is Linux only, actually metalctl also supports macos and windows. We should at least spit out a warning that this command is not available on OSes other than linux. 

On linux a check before if /dev/net/tun is present and give a warning that tun.ko must be loaded.

Reply:
`/var/lib` looks like was an excessive thing. It was used for storing tailscale state, therefore mounting host directory looks unnecessary. Removed that part, at least during testing it didn't affect anything.